README.md

```diff
@@ -38,3 +47,22 @@ Handwrite can generate custom font of your handwriting based on a writing sample
 `python setup.py test`
 
 You are ready to go!
+
+# Usage
+
+## Creating your Handwritten Sample 
+
+1. Take a printout of `handwrite_sample.pdf`, present in the cloned `handwrite` folder.
+2. Fill the form. (Take a look at `handwrite/temp_sheets/sheet_new.jpg` as an example)
+3. Scan the filled form using a scanner, or Adobe Scan in your phone.
+4. Add the jpg image to the `temp_sheets` folder.
+
+Your form should look like this:
+![Filled form](temp_sheets/sheet_new.jpg)
+
+## Creating your font
+1. Activate your environment and go to the cloned folder.
```

Comment:
Neither environment needed nor cloning. Make sure handwrite, potrace and fontforge are installed.